packages/atomic/package.json

```diff
@@ -104,13 +102,14 @@
     "monaco-editor-webpack-plugin": "6.0.0",
     "ncp": "2.0.0",
     "patch-package": "^6.4.7",
-    "postcss-focus-visible": "6.0.1",
+    "postcss-focus-visible": "6.0.4",
     "postcss-import": "14.0.2",
-    "postcss-mixins": "8.1.0",
+    "postcss-mixins": "9.0.1",
     "postcss-nested": "5.0.6",
     "puppeteer": "9.1.1",
     "react": "16.14.0",
-    "rollup-plugin-html": "0.2.1"
+    "rollup-plugin-html": "0.2.1",
+    "tailwindcss": "3.0.19"
```

Comment:
not sure why it was a dependency ?

Reply:
Well, hum, I think it can indeed be seen as a dev dependency, because it's not actually "shipped" to our clients, like they won't actually download and install tailwind when they `npm install @coveo/atomic`